nuxeo-features/nuxeo-elasticsearch/nuxeo-elasticsearch-core/src/test/resources/pageprovider-test-contrib.xml

```diff
@@ -268,7 +268,7 @@
       <property name="coreSession">#{documentManager}</property>
       <property name="maxResults">10000</property>
       <pageSize>20</pageSize>
-      <pattern>SELECT * FROM Document</pattern>
+      <pattern>SELECT * FROM Document WHERE ecm:isCheckedInVersion = 0</pattern>
```

Comment:
`ecm:isCheckedInVersion` has been (not formally) deprecated for a long time, you can use `ecm:isVersion` instead.